GitUp/Application/Base.lproj/MainMenu.xib

```diff
@@ -278,6 +278,120 @@ You must close and reopen any opened repositories in GitUp after changing this s
+                                                </binding>
+                                            </connections>
+                                        </popUpButton>
+                                        <textField horizontalHuggingPriority="251" verticalHuggingPriority="750" id="ep8-Hv-BTg">
+                                            <rect key="frame" x="12" y="346" width="120" height="17"/>
+                                            <autoresizingMask key="autoresizingMask" flexibleMaxX="YES" flexibleMinY="YES"/>
+                                            <textFieldCell key="cell" scrollable="YES" lineBreakMode="clipping" allowsUndo="NO" sendsActionOnEndEditing="YES" alignment="right" title="External Merge:" id="3X7-YE-FqO">
+                                                <font key="font" metaFont="system"/>
+                                                <color key="textColor" name="labelColor" catalog="System" colorSpace="catalog"/>
+                                                <color key="backgroundColor" name="controlColor" catalog="System" colorSpace="catalog"/>
+                                            </textFieldCell>
+                                        </textField>
+                                        <popUpButton verticalHuggingPriority="750" id="wIC-YU-dW2">
+                                            <rect key="frame" x="136" y="340" width="165" height="26"/>
+                                            <autoresizingMask key="autoresizingMask" flexibleMaxX="YES" flexibleMinY="YES"/>
+                                            <popUpButtonCell key="cell" type="push" title="FileMerge" bezelStyle="rounded" alignment="left" lineBreakMode="truncatingTail" borderStyle="borderAndBezel" imageScaling="proportionallyDown" inset="2" autoenablesItems="NO" altersStateOfSelectedItem="NO" selectedItem="MFB-RI-wuw" id="lxS-vb-flN">
+                                                <behavior key="behavior" lightByBackground="YES" lightByGray="YES"/>
+                                                <font key="font" metaFont="menu"/>
+                                                <menu key="menu" autoenablesItems="NO" id="isf-fV-z4D">
+                                                    <items>
+                                                        <menuItem title="FileMerge" id="MFB-RI-wuw"/>
+                                                        <menuItem title="Kaleidoscope" id="AO6-Py-2o5"/>
+                                                        <menuItem title="Beyond Compare" id="Cuo-4H-a9H"/>
+                                                        <menuItem title="P4Merge" id="O2n-kY-CrU"/>
+                                                        <menuItem title="DiffMerge" id="N6F-br-PGJ"/>
+                                                        <menuItem isSeparatorItem="YES" id="rLZ-QI-beC"/>
+                                                        <menuItem title="Git Tool" id="09q-g5-uZM"/>
```

Comment:
Why are these being added? Where were these created before?

Reply:
@lucasderraugh 
Ah, right. I created new tab "Tools" in preferences window.

Reply:
@lucasderraugh 
Before diff-tools options (External Diff / External Merge) were in "Advanced" tab, but it is too clumsy now with additional Terminal option.
I decided to move them to new tab "Tools". You can check it and decide if it is needed.